JavaAssignments/src/saket/Arthmaticoperations.java

```diff
@@ -0,0 +1,49 @@
+package saket;
+
+public class Arthmaticoperations {
+
+	public static void main(String[] args) {
+
+		int a = Integer.parseInt(args[0]);
+		int b = Integer.parseInt(args[1]);
+
+		sum(a, b);
+		sub(a, b);
+		mul(a, b);
+		div(a, b);
+		rem(a, b);
+
+	}
+
+	static void sum(int a, int b) {
```

Comment:
Best practice is to return answer from sum method, capture it in main method and main method should print sum.
For now i am approving this.